Library/Formula/imply.rb

```diff
@@ -0,0 +1,21 @@
+class Imply < Formula
+  desc "Exploratory Analytics Powered By Druid"
+  homepage "http://imply.io/"
+  url "http://static.imply.io/release/imply-#{version}.tar.gz"
+  version "1.1.0"
+  sha256 "d06add07832c4253f0f3722b1625d034c52f3d9041e59779a277b72a84d1ce53"
+
+  bottle :unneeded
+
+  def install
+    libexec.install Dir["*"]
+    bin.write_exec_script Dir["#{libexec}/bin/*"]
+    ohai "Start up services with:"
```

Comment:
Please don't `ohai`. People should check upstream for general usage unrelated to Homebrew.